site/src/ThemeSelector.jsx

```diff
@@ -3,22 +3,21 @@ import { useState, useEffect } from "react";
 import LayerIcon from "./icons/icon-layers.svg?react";
 import "./ThemeSelector.css"; 
 
-function ThemeSelector({ interactiveLayers }) {
+function ThemeSelector({ visibleThemes }) {
 
   const [places, setPlaces] = useState(true);
```

Comment:
Would prefer to have a data-driven way to generate list of layers/types out of themes - could change between releases

Reply:
I 100% agree. Seems like the release notes at https://labs.overturemaps.org/data/releases.json are a good starting point, but need to be augmented to include this information. Avoiding hardcoding this information will also make it possible to toggle between different versions of the data more easily.

Not sure how we can both have dynamic layers and consistent styling though. The `ThemeTypeLayer` includes both the layers/types and styling. I don't think the latter should live in any release manifest.

Reply:
Could we include the entire `schema` repo as a submodule or use the YAMLs as part of the build or dynamically?

The YAML can give us 1) all themes, 2) all types for each theme, and 3) all geometry types in each theme. 
We would have to assign colors, etc in some external way

Reply:
Yes, I love the idea of using the schema directly. I had a similar thought for how we could augment the inspector panel with documentation tooltips: https://github.com/OvertureMaps/io-site/issues/40.

Since Overture releases are versioned, it may make more sense to include the schema as a sidecar for each release. On initialization, we could download the schema, initialize all the layers using the geometry types, and build indices for documentation, etc. This approach would decouple the `io-site` release with Overture data releases, and would enable side-by-side comparison of different versions.